pkg/reconciler/shared/tektonconfig/tektonconfig.go

```diff
@@ -108,6 +108,8 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, tc *v1alpha1.TektonConfi
 		return nil
 	}
 
+	tc.SetDefaults(ctx)
```

Comment:
@vdemeester when we pass the object through defaulting in reconciler, do we also update the object in etcd?

Reply:
i think the defaulter won't do it. however, the next update action down in the code/reconciler will get it into etcd. 
so your concern is right.

The change in this patch should be moved to [here](https://github.com/tektoncd/operator/blob/main/pkg/reconciler/shared/tektonconfig/instance.go#L122)

Reply:
in fact, we can do it in both places, it should have the same effect.

Reply:
im fine with merging this as it is.

Reply:
addressed the comment btw

Reply:
We have the same in `tektoncd/pipeline`, as @nikhil-thomas said, it won't be stored in `etcd` but ain't such a big deal.